packages/flutter_adaptive_scaffold/test/adaptive_scaffold_test.dart

```diff
@@ -863,6 +863,18 @@ void main() {
       expect(rail.labelType, NavigationRailLabelType.all);
     },
   );
+
+  testWidgets('AdaptiveScaffold builds without throwing an error', (WidgetTester tester) async {
```

Comment:
How is this test related to the PR?

There needs to be a test that exercises the new functionality the PR adds.